src/context/basket.js

```diff
@@ -0,0 +1,19 @@
+import React, { createContext } from 'react'
+
+export default BasketProvider
+
+export const BasketContext = createContext({
+    basket: '',
+    updateBasket: () => null
+})
+
+const BasketProvider = ({children}) => {
+    [basket, setBasket] = useState ('')
+
+    return (
+        <BasketContext.Provider value={{basket, setBasket: updateBasket}}>
```

Comment:
setBasket and updateBasket are backwards here

Reply:
swapped them around so should work.